build/charts/antrea/templates/crds/clusternetworkpolicy.yaml

```diff
@@ -224,6 +225,16 @@ spec:
                                   type: integer
                                   minimum: 0
                                   maximum: 255
+                            igmp:
+                              type: object
+                              properties:
+                                igmpType:
+                                  type: integer
```

Comment:
Could you check if we are able to use `enum` to define all possible values? OpenAPI support `enum`, but not sure it is supported with K8s or not.

Reply:
it supports enum, but I did not find a way to describe what does each value mean.

Reply:
done